go-controller/pkg/cni/ovs.go

```diff
@@ -233,20 +238,69 @@ func doPodFlowsExist(queries []openflowQuery) bool {
 	return true
 }
 
-func waitForPodInterface(ctx context.Context, mac string, ifAddrs []*net.IPNet, ifaceName, ifaceID string, ofPort int, checkExternalIDs bool) error {
+// checkCancelSandbox checks that this sandbox is still valid for the current
+// instance of the pod in the apiserver. Sandbox requests and pod instances
+// have a 1:1 relationship determined by pod UID. If we detect that the pod
+// has changed either UID or MAC terminate this sandbox request early instead
```

Comment:
Should we check pod UID, MAC, or IP here?

Reply:
Don't need IP since MAC serves the purpose; it's tied to the IP anyway and won't be re-used.